steam_app/controllers/News/NewsViewController.swift

```diff
@@ -9,11 +9,15 @@ import UIKit
 
 class NewsViewController: UIViewController {
     @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var search: UISearchBar!
     let  newsList = LocalCache.newsList
+    var filteredData = [News]()
```

Comment:
it's a bad approach to put an instance of the Game inside the array as the array will never be empty.
Better to put it this way var filteredData: [News] = []